src/hooks/Circles/usePrivatePosts.ts

```diff
@@ -170,11 +175,11 @@ export const useLookupUsers = (userIds: string[]) => {
   };
 
   return useQueries({
-    queries: (userIds ?? []).map((userId) => {
+    queries: (userList ?? []).map((user) => {
       return {
-        queryKey: ['user', userId],
-        queryFn: () => queryUser(userId),
-        enabled: !!accountHeaders?.['LifeOmic-Account'],
+        queryKey: ['user', user.userId],
+        queryFn: () => queryUser(user.userId),
```

Comment:
Does this combine multiple requests or do we do N network requests for N users?

Reply:
We do N network requests for N users based on which users have the enabled flag set to true.